subprojects/internal-instrumentation-processor/src/main/java/org/gradle/internal/instrumentation/processor/codegen/CompositeInstrumentationCodeGenerator.java

```diff
@@ -46,7 +46,7 @@ public GenerationResult generateCodeForRequestedInterceptors(Collection<CallInte
         }
 
         List<CanGenerateClasses> generatingResults = results.stream().map(it -> (CanGenerateClasses) it).collect(Collectors.toList());
-        Map<String, CanGenerateClasses> generatorByClassName = new HashMap<>();
+        Map<String, CanGenerateClasses> generatorByClassName = new LinkedHashMap<>();
```

Comment:
❓ I assume the order of the `generatingResults` collection is stable as well?

Reply:
Yes, it is, as long as all the generator implementations produce the results in a stable order.